include/zephyr/bluetooth/hci_types.h

```diff
@@ -37,6 +37,14 @@ extern "C" {
 #define BT_HCI_SYNC_HANDLE_INVALID 0xffff
 #define BT_HCI_PAWR_SUBEVENT_MAX 128
 
+/* Bluetooth spec v5.4 Vol 4, Part E - 5.4.3 HCI Synchronous Data Packets */
+struct bt_hci_sco_hdr {
+	uint16_t conn_handle;
```

Comment:
Ok, I just checked and the HCI spec uses `Connection_Handle` both for SCO and ISO, but just `Handle` for ACL. However, since the ISO struct already uses simply `handle` I'd follow that convention for SCO as well.

Reply:
changed